ports/wxwidgets/vcpkg.json

```
@@ -35,10 +34,7 @@
     "nanosvg",
     "opengl",
     "pcre2",
```

This is what you should change!
```suggestion
    {
      "name": "pcre2",
      "default-features": false
    },
```